docs/Project-Configuration/Evergreen-Data-for-Analytics.md

```diff
@@ -58,6 +58,151 @@ Daily aggregated statistics for test executions run in Evergreen. Test stats are
+| depends\_on                    | ROW       | depends_on row describing the task's dependencies.
+| num\_dependents                | BIGINT    | The number of tasks this task depends on.
+| override_dependencies          | BOOLEAN   | Whether this task will wait on its dependencies.
+| is\_essential\_to\_succeed     | BOOLEAN   | Indicates that this task must finish in order for its build and version to be considered successful. For example, tasks selected by the GitHub PR alias must succeed for the GitHub PR requester before its build or version can be reported as successful, but tasks manually scheduled by the user afterwards are not required.
+| display\_task\_id              | VARCHAR   | ID of the task's display task.
+| display\_task\_display\_name   | VARCHAR   | Display name of the task's display task.
+| parent\_patch\_id              | VARCHAR   | Patch ID of the task's parent patch, if the task is a display task.
+| parent\_patch\_number          | BIGINT    | Order number of the task's parent patch, if the task is part of a child patch.
+| display\_only                  | VARCHAR   | Whether this task is a display task
+| execution\_tasks               | ARRAY     | IDs of the execution tasks of this task, if the task is a display task.
+| activated\_by                  | VARCHAR   | User or service that activated this task.
+| create\_time                   | TIMESTAMP | The creation time for the task, derived from the commit time or the patch creation time.
+| ingest\_time                   | TIMESTAMP | Time the task was created.
+| container\_allocated_time      | TIMESTAMP | Time a container was allocated to run the task, if the task is a containerized task.
+| dispatch\_time                 | TIMESTAMP | Time the scheduler assigns the task to a host.
+| scheduled\_time                | TIMESTAMP | Time the task is first scheduled.
+| start\_time                    | TIMESTAMP | Time the task started running.
+| finish\_time                   | TIMESTAMP | Time the task completed running.
+| activated_time                 | TIMESTAMP | Time the task was marked as available to be scheduled, automatically or by a developer.
+| dependencies\_met\_time        | TIMESTAMP | Time all dependencies are met, for tasks that have dependencies.
+| last\_heartbeat                | TIMESTAMP | Time of the last heartbeat sent back by the agent.
+| time\_taken                    | BIGINT    | Duration, in nanoseconds, the task took to execute, if it has finished, or how long the task has been running, if it has started.
+| wait\_since\_dependencies\_met | BIGINT    | Duration, in nanoseconds, since the task's dependencies have been met.
+| duration\_prediction           | BIGINT    | How long Evergreen estimates this task will take to run.
+| distro                         | VARCHAR   | ID of the distro this task is for.
+| distro\_aliases                | ARRAY     |  Optional secondary distros that can run the task when they're idle.
```

Comment:
This is renamed to `secondary_distros` in the view. 
We may want to consider specifying the array type: `ARRAY(VARCHAR)`.

Nit: Extra whitespace before "Optional".

Reply:
fixed